src/components/Q&A/QuestionComp.jsx

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import ReactDOM from "react-dom";
+import AddAnswer from './AddAnswer.jsx';
+import {API_KEY} from '../../config/config.js';
+const axios = require('axios');
+
+class QuestionComp extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      questions: this.props.questionList,
+      helpClick: false,
+      showAns: false
+    }
+    this.handleQHelp = this.handleQHelp.bind(this);
+    this.handleClose = this.handleClose.bind(this);
+  }
+
+
+  handleQHelp(qObj) {
+    var temp = parseInt(qObj.question_id);
```

Comment:
I think this variable name could be changed to something that would better reflect what its intent (like questionIDNum??? )